src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngEntityCommand.php

```diff
@@ -503,7 +503,11 @@ public function getRedirectUrls()
      */
     public function setRedirectUrls($redirectUrls)
     {
-        $this->redirectUrls = $redirectUrls;
+        $urls = []; // because numeric array keys can be sparse so a for-loop cannot be trusted
+        foreach ($redirectUrls as $url) {
+            $urls[] = strtolower($url);
+        }
```

Comment:
Maybe an `array_walk` solution is a bit nicer here. But this works well too!

Alternatively to ensure you preserve the keys you could iterate over the urls with a key->value construction.

```
foreach ($redirectUrls as $key => $url) {
    $urls[$key] = strtolower($url);
}

Reply:
I actually used this because i didn't want to preserve the keys.  It's weird behaviour for it to not reorder the keys.  Can lead to subtle bugs & given that they are numeric (and they're not used) there's no reason for us not to reorder.